setup.py

```diff
@@ -54,16 +54,17 @@
     include_package_data=True,
     install_requires=[
         'bumpversion>=0.5.3,<1',
-        'eth-abi>=1.2.2,<1.3.0',
         'eth-keys>=0.2.0b3,<1',
         'eth-utils>=1.3.0,<2',
+        'eth-tester>=0.1.0b34,<1',
         'ipfsapi>=0.4.3,<1',
         'jsonschema>=2.6.0,<3',
         'protobuf>=3.0.0,<4',
-        'pytest-ethereum>=0.1.3a.5,<1',
+        'pytest-ethereum>=0.1.3a.6,<1',
+        'py-evm>=0.2.0a38,<1',
         'py-solc>=3.2.0,<4',
         'rlp>=1.0.1,<2',
-        'web3[tester]>=4.7,<5',
+        'web3[tester]>=5.0.0a2,<6.0.0',
```

Comment:
Is this a testing dependency?

Reply:
@pipermerriam The `[tester]` splinter is a testing dependency, but `web3` by itself is needed for a user to use `py-ethpm`. 

I tried moving `web3[tester]>=5.0.0a3,<6.0.0` to the `test` section of `extras_require` and leaving `web3>=5.0.0a3,<6.0.0` in the `install_requires` section, in the hopes that the `[tester]` splinter would only be installed with `[test]` or `[dev]`. But this failed to install `eth_tester` when running `pip install -e .[dev]`. 

Is there a better way to handle only installing the `[tester]` splinter in a the `extras_require[test]` env, but the same `web3` version without the `[tester]` splinter in `install_requires`?